test/chunkify/chunkify_test.js

```diff
@@ -23,70 +23,61 @@ describe('Chunkify', function () {
     });
 
     /**
-     * Tests whether valid message within a specified CHAR_LIMIT is un-modified.
+     * Tests whether randomized valid message within a specified CHAR_LIMIT is un-modified.
      */
 
-    it('should not modify valid simple messages', function(done) {
+    it('should not modify valid simple messages (randomized input)', function(done) {
         const actualChunkifiedMessage = chunkify("Valid Message", TELEGRAM_CHAR_LIMIT);
         assert.deepEqual(actualChunkifiedMessage, ["Valid Message"]);
         done();
     });
 
     /**
-     * Tests whether small valid messages containing CHAR_LIMIT characters is un-modified.
+     * Tests whether randomized, small valid messages containing CHAR_LIMIT characters is un-modified.
      */
 
-    it('should not modify valid small messages', function(done) {
+    it('should not modify valid small messages (randomized input)', function(done) {
         const VALID_SMALL_TEST_MESSAGE = randomTextGenerator(SMALL_CHAR_LIMIT);
         const actualChunkifiedMessage = chunkify(VALID_SMALL_TEST_MESSAGE, SMALL_CHAR_LIMIT);
         assert.deepEqual(actualChunkifiedMessage, [VALID_SMALL_TEST_MESSAGE]);
         done();
     });
 
     /**
-     * Tests whether large valid messages containing CHAR_LIMIT characters is un-modified.
+     * Tests whether randomized, large valid messages containing CHAR_LIMIT characters is un-modified.
      */
 
-    it('should not modify valid large messages', function(done) {
+    it('should not modify valid large messages (randomized input)', function(done) {
         const VALID_LARGE_TEST_MESSAGE = randomTextGenerator(TELEGRAM_CHAR_LIMIT);
         const actualChunkifiedMessage = chunkify(VALID_LARGE_TEST_MESSAGE, TELEGRAM_CHAR_LIMIT);
         assert.deepEqual(actualChunkifiedMessage, [VALID_LARGE_TEST_MESSAGE]);
         done();
     });
 
     /**
-     * Tests whether large messages containing number of characters greater than CHAR_LIMIT characters are modified appropriately
-     * (i.e. no Empty Strings within output).
+     * Tests whether randomized, large messages containing number of characters greater than CHAR_LIMIT characters
+     * are modified appropriately (i.e. no Empty Strings within output).
      */
 
-    it('should split large messages in order of linebreaks, whitespace and characters where appropriate', function(done) {
+    it('should split large messages in order of linebreaks, whitespace and chars where appropriate (randomized input)', function(done) {
```

Comment:
This test doesn't check for the order though! It only verifies that the length is within certain bounds. I might write "should return an array of strings with strictly positive size less than the char limit"

Reply:
Updated per specifications